omnigibson/controllers/multi_finger_gripper_controller.py

```diff
@@ -163,7 +163,8 @@ def compute_control(self, goal_dict, control_dict):
         # Choose what to do based on control mode
         if self._mode == "binary":
             # Use max control signal
-            if target[0] >= 0.0:
+            should_open = target[0] >= 0.0 if not self._inverted else target[0] > 0.0
```

Comment:
what's the difference here? should this be < instead of >?

Reply:
If `inverted` is True, the target is already "inverted", but the only edge case is the exact value of 0.0, which should mean "open". But before my changes, 0.0 will mean "closed".